app.css

```diff
@@ -1,34 +1,80 @@
-body {
+html  {
+  height: 100%;
+  width: 100%;
   font-size: 1rem;
   font-family: 'Cabin', sans-serif;
+  text-align: center;
```

Comment:
Only center the things you need centered. Bootstrap has a helper class for this.